src/Pelagos/Bundle/AppBundle/Controller/UI/DataDiscoveryController.php

```diff
@@ -40,22 +38,22 @@ public function defaultAction(Request $request)
             array(
                 'defaultFilter' => $request->query->get('filter'),
                 'pageName' => 'data-discovery',
-                'download' => false,
```

Comment:
We use the dangling comma, we use it in other places, so we have to be consistent.